zebedee/client.go

```diff
@@ -282,6 +282,14 @@ func (c *Client) GetTimeseriesMainFigure(ctx context.Context, userAccessToken, c
 	return ts, nil
 }
 
+func (c *Client) GetResourceBody(ctx context.Context, userAccessToken, collectionID, lang, uri string) ([]byte, error) {
```

Comment:
Exported funcs should have godoc
```suggestion
// GetResourceBody returns …
func (c *Client) GetResourceBody(ctx context.Context, userAccessToken, collectionID, lang, uri string) ([]byte, error) {
```